src/main/java/morty/TaskList.java

```diff
@@ -0,0 +1,101 @@
+package morty;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import morty.task.Task;
+
+public class TaskList {
+
+  /**
+   * The list of tasks.
+   */
+  private List<Task> tasks;
+
+  /**
+   * Constructs a TaskList object.
+   */
+  public TaskList() {
+    tasks = new ArrayList<Task>();
+  }
+
+  /**
+   * Constructs a TaskList object with the given list of tasks.
+   *
+   * @param tasks The list of tasks.
+   */
+  public TaskList(List<Task> tasks) {
+    this.tasks = tasks;
+  }
+
+  /**
+   * Returns the task at the given index.
+   *
+   * @param index The index of the task.
+   * @return The task at the given index.
+   */
+  public Task get(int index) {
+    return tasks.get(index);
+  }
```

Comment:
Perhaps this method could be renamed `getTask` or something similar for clarity on its usage?